### PR TITLE
Inconsistent Rounding

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

Using .toFixed() for rounding seems to cause a weird issue were it only rounds half the time, so I'm switching to good ol' shift forward by two decimals, round and then shift back by two decimals. We've all seen and hate this, if anyone knows a better way, please let me know!

[This explains the issue well](https://stackoverflow.com/questions/77144696/seemingly-inconsistent-tofixed-behavior). Scroll to the **toFixed(1) example** example

### Related ticket(s)

CMDCT-3780

---
### How to test

1) Sign into QMR, any separate user. i.e. [stateuser1@test.com](mailto:stateuser1@test.com)
2) Enter ADD-CH: Medicaid
3) Fill out Measure Specification to trigger the Performance Measure layer
4) For the Data Source: Select Administrative Data.
5) In the Performance Measure Section:
   - Enter Numerator: 305 
   - Enter Denominator: 10000
   - The Rate should generate to 3.1 (actual rate would be 3.05 but rounded to 3.1)
6) Save
7) Do steps 3 - 5 for ADD-CH: CHIP
8) Go to the Combined Rates Page for this Measure. It should look like this
Screenshot 2024-07-09 at 4 44 09 PM

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — APPROVED** [review]
Works great in the deployed environment.

**Review by benmartin-coforma — APPROVED** [review]
I'd like to know what the original author intended the `+ Number.EPSILON` to accomplish, because I'm not convinced it needs to be there. But, for the purposes of this PR, we'd better leave it alone.

Approved!

File: services/app-api/utils/constants/math.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//this function is copied from in ui-src/utils/rateFormulas/index.ts, we should merge this into a shared file if we ever figure it out
+export const fixRounding = (value: number, numbersAfterDecimal: number) => {
+  return (
+    Math.round((value + Number.EPSILON) * Math.pow(10, numbersAfterDecimal)) /
```
Comment (rocio-desantiago) [inline services/app-api/utils/constants/math.ts:4]: Oooh `Number.EPSILON` - first time seeing this one. I see that it "represents the difference between 1 and the smallest floating point number greater than 1." - per MDN web docs. 

And it's value is `2.220446049250313e-16`. Clarifying question, is the purpose of adding Number.EPSILON to round to the nearest decimal point for accuracy?

**Review by rocio-desantiago — COMMENTED** [review]
Question about Number.EPSILON

File: services/app-api/utils/constants/math.ts (hunk 1 above)
Comment (ailZhou) [reply services/app-api/utils/constants/math.ts:4]: not in this instance, its the .toFixed(1) that does the rounding. this is basically a copy over of what we already have that the client approved of for the rate cals in the front end

Comment (rocio-desantiago) [reply services/app-api/utils/constants/math.ts:4]: Gotcha 👍

**Pull request merged**